### PR TITLE
add LVGL-Rlottie interface as 3rd party lib

### PR DESCRIPTION
**Pull request by kisvegabor**
### Description of the feature or fix

See https://github.com/lvgl/lvgl/issues/2276#issuecomment-946628803

### Checkpoints
- [ ] Follow the [styling guide](https://github.com/lvgl/lvgl/blob/master/docs/CODING_STYLE.md)
- [ ] Update [CHANGELOG.md](https://github.com/lvgl/lvgl/blob/master/docs/CHANGELOG.md)
- [ ] Update the documentation

### CODE REVIEW

**Comment by xiaoxiang781216** [comment]
Look like the string exceed the compiler limitation?
```
CMakeFiles/lvgl_examples.dir/home/runner/work/lvgl/lvgl/examples/libs/rlottie/lv_rlottie_example_card.c.o
Error: /home/runner/work/lvgl/lvgl/examples/libs/rlottie/lv_rlottie_example_card.c:4:3: error: string length ‘34388’ is greater than the length ‘4095’ ISO C99 compilers are required to support [-Woverlength-strings]
```

**Comment by kisvegabor** [comment]
Wow, I didn't know that there is such a limitation. I've converted the string to hex array. It should work this way too.

**Review by xiaoxiang781216 — APPROVED** [review]

File: lv_conf_template.h [hunk 1]
```diff
@@ -561,6 +561,9 @@ e.g. "stm32f769xx.h" or "stm32f429xx.h"*/
 # define LV_FREETYPE_CACHE_SIZE (16 * 1024)
 #endif
 
+/*Rlottie library*/
+#define LV_USE_RLOTTIE 0
```
Comment (xiaoxiang781216) [inline lv_conf_template.h:568]: need update Kconfig too and lv_conf_internal.h need regenerate again

Comment (embeddedt) [reply lv_conf_template.h:568]: Not related specifically to this PR, but I wonder if I should make a CI check at some point that fails if the version of `lv_conf_internal.h` on the commit does not match the output of `scripts/lv_conf_internal_gen.py` from that commit.

Comment (xiaoxiang781216) [reply lv_conf_template.h:568]: Good idea.

**Comment by kisvegabor** [comment]
Before merging it should be tested. Can you give me some instructions about what to add to a main CMakeFile to make it compile? I'm using the [lv_sim_eclipse_sdl](https://github.com/lvgl/lv_sim_eclipse_sdl) project.

**Comment by kisvegabor** [comment]
I followed the instruction from rlottie's README and built and installed it with 
```
make 
make install
```

It works well. Some platform specific description about the integration would be great but it can be added later. 

Please review the changes again.

**Review by xiaoxiang781216 — COMMENTED** [review]
@kisvegabor I add LV_USE_RLOTTIE to Kconfig and squash all patch into one, the change look fine to me now.

**Comment by kisvegabor** [comment]
Great, thanks.

**Pull request merged**